File: 217. Contains Duplicate.cpp
#include <unordered_map>
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map <int, int> dic;
        for(int i = 0; i < nums.size(); i++)
        {
            if (dic.count(nums[i]) > 0)
                return true;
            dic[nums[i]] = 1;
        }
        return false;
    }
};

File: 217. Contains Duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        dic = {}
        for i in nums:
            if i not in dic:
                dic[i] = 1
            else:
                return True
        return False
